packages/devtools/lib/src/memory/memory_plotly.dart

```diff
@@ -409,6 +407,16 @@ class MemoryPlotly {
 class EventTimeline {
   EventTimeline(this._domName, this._chart);
 
+  // Light blue and Dark Blue 600 dark mode colors see
+  // https://standards.google/guidelines/google-material/color/dark-theme.html#style
+  static String snapshotColor =
+      colorToCss(const ThemedColor(Color(0xFF0000FF), Color(0xFF185AE1)));
+  static String resetColor =
+      colorToCss(const ThemedColor(Color(0xFF0000FF), Color(0xFF185AE1)));
+  // Light is Browser's 'lightblue' color, Dark is Blue 300 dark mode colors.
+  static String eventBgColor =
+      colorToCss(const ThemedColor(Color(0xFFABD2DF), Color(0xFF79B6F6)));
+
```

Comment:
can we make these vars ThemedColors and then use colorToCss inline where they are used? This will keep this code consistent with the rest of the color definitions in devtools

Reply:
I did that.  However, colors that are only used in plotly more than once would be more efficient to only call colorToCss once not N times.